src/libCli/Call.cpp

```diff
@@ -193,54 +193,68 @@ int call(ParsedElement & parseTree)
         return -1;
     }
 
-    if(method->client_streaming())
-    {
-        std::cerr << "Error: Client streaming RPCs not supported." << std::endl;
-        return -1;
-    }
+    //if(method->client_streaming())
+    //{
+    //    std::cerr << "Error: Client streaming RPCs not supported." << std::endl;
+    //    return -1;
+    //}
 
     const grpc::protobuf::Descriptor* inputType = method->input_type();
 
     // now we have to construct a protobuf from the parsed argument, which corresponds to the inputType
     google::protobuf::DynamicMessageFactory dynamicFactory;
 
-    // read data from the parse tree into the protobuf message:
-    std::unique_ptr<grpc::protobuf::Message> message = cli::parseMessage(parseTree, dynamicFactory, inputType);
-
+    std::vector<ArgParse::ParsedElement*> requestMessages;
+    parseTree.findAllSubTrees("Message", requestMessages, true, 2);
```

Comment:
why 2?

Reply:
actually this has never worked -> good catch :+1:
I fixed the findAllSubTrees(...) function.
Additionally I found out, that I do not need to specify depth here. the boolean flag here prevents searching deeper if a match is found in the tree, which is exactly what we want to prevent recursion here.